paasta_tools/deployd/common.py

```diff
@@ -50,6 +51,7 @@ def __new__(
         failures: int = 0,
         bounce_timers: Optional[BounceTimers] = None,
         priority: Optional[int] = None,
+        reprocessed_count: int = 0,
```

Comment:
What if this was called "processed_count" and started at 0 still, but was incremented on workers.py:115 instead? (where it actually gets processed I think)

Reply:
I think it's okay to modify where it is now because the `process_service_instance` doesn't actually return a `service_instance`.

But I do agree that `processed_count` is clearer let me update